O/OpenBLAS/common.jl

```diff
@@ -147,7 +147,7 @@ function openblas_script(;num_64bit_threads::Integer=32, openblas32::Bool=false,
     elif [[ ${target} == arm-* ]]; then
         flags+=(TARGET=ARMV7)
     elif [[ ${target} == powerpc64le-* ]]; then
-        flags+=(TARGET=POWER8)
+        flags+=(TARGET=GENERIC DYNAMIC_ARCH=1)
```

Comment:
Turns out I was wrong. We want `TARGET=POWER8` here. But let me check maybe `generic` is enough